sdk/keyvault/samples/sharelink/KeyVaultStorageModelFactory.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Security.KeyVault.Storage
+{
+    [CodeGenClient("AzureSecurityKeyVaultStorageModelFactory")]
+    internal partial class KeyVaultStorageModelFactory
```

Comment:
Can we just prevent this class from being generated? It just complicates the sample that was supposed to be minimal.

Reply:
If we can't, this should at least be `StorageModelFactory`, though I suppose that conflicts potentially with Storage. I suppose this name would be fine, but I'd rather the type not even be generated. All the generated source is already being *copied* into projects that need this, so they can make anything internal they want to.

Reply:
`KeyVaultStorageModelFactory` generation suppressed